spec/support/shared/examples/metasploit/cache/module/class/namable.rb

```diff
@@ -0,0 +1,9 @@
+shared_examples_for 'Metasploit::Cache::Module::Class::Namable' do
+  context 'associations' do
+    it { is_expected.to have_one(:name).class_name('Metasploit::Cache::Module::Class::Name').dependent(:destroy).inverse_of(:module_class).with_foreign_key(:module_class_id) }
+  end
+
+  context 'validations' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.